hr_holidays_calendar_event_privacy/models/hr_leave.py

```diff
@@ -11,7 +11,7 @@ def _prepare_holidays_meeting_values(self):
         """Set calendar events privacy for hr leaves according to type"""
         res = super()._prepare_holidays_meeting_values()
         for user, holidays in groupby(self, lambda h: h.user_id):
-            for holiday, meeting_vals in zip(holidays, res.get(user.id)):
+            for holiday, meeting_vals in zip(holidays, res.get(user.id), strict=False):
                 meeting_vals[
                     "privacy"
                 ] = holiday.holiday_status_id.calendar_event_privacy
```

Comment:
I face this error when don't set the privacy on Leave Type. So, we should not update the value of `privacy` in this case.

![image](https://github.com/user-attachments/assets/9327fc5d-ca74-4436-b341-7b7f9c433e22)